fixtures/builder.py

```diff
@@ -40,6 +41,15 @@ def __init__(self, name):
 
         self.repository.status()
 
+    def __del__(self):
+        # Create a hash for the generated fixture. Generate the hash before
+        # we open the `hash.txt` file because the empty file will affect the
+        # hash of directory.
+        hashes = 'client:' + dirhash(self.dir + '/client', 'sha256')
+        hashes += "\nserver:" + dirhash(self.dir + '/server', 'sha256')
```

Comment:
I just realized that when we were just hashing `self.dir` that the hash would also be affected by the `__init__.py` files themselves, even if the changes didn't make any changes to the fixtures.  So if we add/updated comments or var names this would change the hash. So are reviewer to do a thorough job in these cases they would have to actually compare the generated fixtures to make sure they **hadn't actually changed**.

Also if we wanted to add README files to the fixtures this would change the hashes. 

So I think it is better only hash the `client` and `server` directories. I also putting them on separate lines. I think this will make review easier also. In the case where there are only expected changes to the server files this will prove that without the reviewer having to manually check the generated `client` folder.

Reply:
That's a good point. I agree with your reasoning and I think this change looks good.